chat-client/src/client/chat.ts

```diff
@@ -10,42 +10,51 @@ import {
     InsertToCursorPositionParams,
     SEND_TO_PROMPT,
     SendToPromptMessage,
-    TAB_ID_RECEIVED,
-    TabIdReceivedParams,
     UiMessage,
 } from '@aws/chat-client-ui-types'
 import {
+    CHAT_REQUEST_METHOD,
     ChatParams,
+    FEEDBACK_NOTIFICATION_METHOD,
+    FOLLOW_UP_CLICK_NOTIFICATION_METHOD,
     FeedbackParams,
     FollowUpClickParams,
+    INFO_LINK_CLICK_NOTIFICATION_METHOD,
     InfoLinkClickParams,
+    LINK_CLICK_NOTIFICATION_METHOD,
     LinkClickParams,
+    QUICK_ACTION_REQUEST_METHOD,
+    QuickActionParams,
+    READY_NOTIFICATION_METHOD,
+    SOURCE_LINK_CLICK_NOTIFICATION_METHOD,
     SourceLinkClickParams,
+    TAB_ADD_NOTIFICATION_METHOD,
+    TAB_CHANGE_NOTIFICATION_METHOD,
+    TAB_REMOVE_NOTIFICATION_METHOD,
     TabAddParams,
     TabChangeParams,
     TabRemoveParams,
 } from '@aws/language-server-runtimes-types'
-import {
-    CHAT_PROMPT,
-    FEEDBACK,
-    FOLLOW_UP_CLICKED,
-    INFO_LINK_CLICK,
-    LINK_CLICK,
-    NEW_TAB_CREATED,
-    SOURCE_LINK_CLICK,
-    ServerMessage,
-    TAB_CHANGED,
-    TAB_REMOVED,
-    TELEMETRY,
-    TelemetryParams,
-    UI_IS_READY,
-} from '../contracts/serverContracts'
+import { MynahUIDataModel } from '@aws/mynah-ui'
+import { ServerMessage, TELEMETRY, TelemetryParams } from '../contracts/serverContracts'
 import { ENTER_FOCUS, EXIT_FOCUS } from '../contracts/telemetry'
 import { Messager, OutboundChatApi } from './messager'
 import { InboundChatApi, createMynahUi } from './mynahUi'
 import { TabFactory } from './tabs/tabFactory'
 
-export const createChat = (clientApi: { postMessage: (msg: UiMessage | ServerMessage) => void }) => {
+const DEFAULT_TAB_DATA = {
```

Comment:
Is the plan to move these properties to `InitializeResult` too?

Reply:
Yes, would be nice to have

Reply:
Can we please have a task for this?
Otherwise, `DEFAULT_TAB_DATA` better fits into `TabFactory` if we are going to leave it hardcoded.